index.js

```diff
@@ -204,11 +204,24 @@ function markdownTable(table, options) {
     rule = [];
 
     while (++index < cellCount) {
+      /* When `straighten` is false, make the rule the same size as the first row. */
+      if (settings.straighten === false) {
+        value = table[0][index];
+        if (value === null || value === undefined) {
```

Comment:
Could this `if/else` be replaced with a ternary operator? Something like: `value = value ? String(value) : ''`, or even `value = String(value || '')`